### ISSUE TITLE
# FIXME: update winetricks_is_installed to look at installed_file2..n

### ISSUE THREAD

**Issue by austin987** [Winetricks/winetricks#988]
Mentioned above flash_metadata.

We should loop over a reasonable number of installed files instead of only checking 1. I'd say 25 is probably a safe limit, since a file existence check is pretty cheap and I doubt we'll ever need more than 5/10.

**Comment by h1z1** [comment]
Can  you elaborate a bit on what you mean here?  Batch downloading or?

**Comment by austin987** [comment]
@h1z1 no. winetricks_is_installed() checks if the file passed to it is present in the WINEPREFIX. This is used to check if a verb has already been installed and to check for success after installing.

This issue is about expanding that function to allow checking for more than one file in the prefix (as flash tries to do, but it's currently a no-op).
